help/learn/Custom-Machine-Learning.md

```diff
@@ -0,0 +1,132 @@
+# Custom ML models
+ 
+ Datagrok supports three ML engines out of the box: H2O, Caret, Chemprop. In addition, the platform allows users to build their own customizable ML models. This capability provides full access and control to algorithms available in ML libraries in any of the supported languages (such as Python, R, and Julia). The users may construct and configure any chosen model in their custom data pipeline.   
```

Comment:
Let's add links to relevant pages: `[supported languages](../develop/scripting.md#supported-languages)`